src/components/AppShell/AppShell.tsx

```diff
@@ -78,9 +79,71 @@ export const SlimContent = ({ children }: { children: ReactNode }) => (
   </LinearGradient>
 );
 
+interface HeaderProps {
+  title?: string;
+  subtitle?: string;
+  fixedHeader?: boolean;
+  alt?: boolean;
+}
+
+export const AppHeader = ({
+  alt,
+  title,
+  subtitle,
+  fixedHeader,
+}: HeaderProps) => {
+  console.log('AppHeader', fixedHeader);
```

Comment:
Whoops. Are we missing eslint that catches this stuff? :)

Reply:
Whoops indeed! Yeah, this project has very basic ``.eslintrc`` and ``.prettierrc`` rules.